src/examples/local-state-key-change.js

```diff
@@ -0,0 +1,74 @@
+// useEffect: persistent state
+// 💯 flexible localStorage hook - changing the key in localStorage
```

Comment:
We don't need 💯 because it's not an extra credit.

```suggestion
// flexible localStorage hook - changing the key in localStorage
```